README.md

```diff
@@ -70,17 +70,20 @@ of  `/etc/resolver/dev.gov.uk`.
 cp /etc/resolver/dev.gov.uk ~/dev.gov.uk
 ```
 
-Then create or append to the following and restart dnsmasq. If you've been using
-the vagrant based dev vm, you'll need to replace `/etc/resolver/dev.gov.uk`..
```

Comment:
@VitalieMogoreanu I think we want to keep this line about replacing the file if you've been using the VM?